modules/auxiliary/scanner/http/dell_idrac.rb

```diff
@@ -114,25 +161,58 @@ def report_cred(opts)
 
   def run_host(ip)
     print_status("Verifying that login page exists at #{ip}")
-    uri = normalize_uri(target_uri.path)
+    server_response = false
     begin
+      # <= v8
       res = send_request_raw({
         'method' => 'GET',
-        'uri' => uri
+        'uri' => pre_v9_url
       })
 
-      if (res && (res.code == 200) && !res.body.to_s.match(/<authResult>1/).nil?)
-        print_status('Attempting authentication')
+      if res && res.code == 200
+        server_response = true
+        if !res.body.to_s.match(/<authResult>1/).nil? || # version <8
+           !res.body.to_s.match(/<authResult>99/).nil? # version 8 of idrac shows 99 on first connect
+          print_status('Attempting authentication against iDRAC version < 9')
+
+          each_user_pass do |user, pass|
+            do_login_pre9(user, pass)
+          end
```

Comment:
Does this need a return here in this logic, to ensure we don't start logging into v9 as well?